readme_CN.md

```diff
@@ -46,33 +48,37 @@ Opik 帮助您构建、评估和优化 LLM 系统，使其运行得更好、更
 
 ## 🚀 什么是 Opik？
 
-Opik（由 [Comet](https://www.comet.com?from=llm&utm_source=opik&utm_medium=github&utm_content=what_is_opik_link&utm_campaign=opik) 开发）是一款开源平台，旨在简化整个 LLM 应用生命周期。它让开发者能够评估、测试、监控和优化模型及智能体系统。主要功能包括：  
-* **全面可观测性**：深度跟踪 LLM 调用、对话日志及智能体活动。  
-* **高级评估**：强大的提示评估、LLM-as-a-judge 及实验管理。  
-* **生产就绪**：可扩展的监控仪表板和在线评估规则。  
-* **Opik Agent Optimizer**：用于提升提示和智能体的专用 SDK 与优化器。  
-* **Opik Guardrails**：帮助您实施安全且负责任的 AI 实践。
+Opik（由 [Comet](https://www.comet.com?from=llm&utm_source=opik&utm_medium=github&utm_content=what_is_opik_link&utm_campaign=opik) 开发）是一款开源平台，旨在简化整个 LLM 应用生命周期。它让开发者能够评估、测试、监控和优化模型及智能体系统。主要功能包括：
+
+- **全面可观测性**：深度跟踪 LLM 调用、对话日志及智能体活动。
+- **高级评估**：强大的提示评估、LLM-as-a-judge 及实验管理。
+- **生产就绪**：可扩展的监控仪表板和在线评估规则。
+- **Opik Agent Optimizer**：用于提升提示和智能体的专用 SDK 与优化器。
```

Comment:
Need to translate the following entries?

Reply:
Good question, it was already not-translated, I will go